CHANGELOG.md

```diff
@@ -104,6 +104,8 @@ and this project adheres to
   - [#1580](https://github.com/iovisor/bpftrace/pull/1580)
 - Printing of small integers with `printf`
   - [#1532](https://github.com/iovisor/bpftrace/pull/1532)
+- Fix buf builtin testcases for big endian
```

Comment:
We try to keep the changelog for end users, so internal test changes are not relevant

Reply:
Ok. Will remove the changelog then. Thanks